src/StatusTable/StatusTable.tsx

```diff
@@ -71,7 +71,7 @@ const StatusTable: FC<StatusTableProps> = (
 
     return (
         <ThemeProvider theme={theme}>
-            <StatusTableStyle className={className} StatusTableCard={false}>
+            <StatusTableStyle className={className} cards>
```

Comment:
Shouldn't this be just `<StatusTableStyle className={className}>` (without the `cards`)?  That's what's semantically equivalent to what you had before.

Reply:
So I realized that I created a duplicate property. We dont need StatusTableCard but instead we can use cards.